### PR TITLE
Upstream merge

### PR DESCRIPTION
**Pull request by drexample**
## About The Pull Request

Decided not to delay too much and get 5000 merge conflicts

## Why It's Good For The Game

More content

## Changelog
:cl:
add: oopstrem merg
/:cl:

### CODE REVIEW

File: code/modules/economy/economy_misc.dm [hunk 1]
```diff
@@ -131,7 +131,7 @@ var/global/datum/computer_file/data/email_account/service/payroll/payroll_mailer
 
 	//create an entry in the account transaction log for when it was created
 	var/datum/transaction/T = new(department.account_initial_balance, department_account.owner_name, "Account creation", "Free Trade Union Terminal #277")
-	T.date = "2 April, 2555"
+	T.date = "2 April, 2339"
```
Comment (EvilJackCarver) [inline code/modules/economy/economy_misc.dm:134]: This date is still in the future - is this intentional?

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1561,6 +1573,14 @@ var/list/rank_prefix = list(\
 	to_chat(src, "<span class='notice'>You are now [pulling_punches ? "pulling your punches" : "not pulling your punches"].</span>")
 	return
 
+/mob/living/carbon/human/verb/toggle_dodging()
+	set name = "Toggle Dodging"
+	set desc = "Just stand still while under fire."
+	set category = "IC"
+	if(stat) return
+	dodging = !dodging
+	to_chat(src, "<span class='notice'>You are now [dodging ? "dodging incoming fire" : "not dodging incoming fire"].</span>")
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/human/human.dm:1582]: `to_chat(src, "<span class='notice'>You are [dodging ? "now dodging incoming fire" : "no longer dodging incoming fire"].</span>")`

would probably work better than "now dodging"/"now not dodging"

File: code/modules/mob/living/carbon/superior_animal/stalker/stalker.dm [hunk 3]
```diff
@@ -0,0 +1,77 @@
+// OneStar patrol borg that defends OneStar facilities
```
Comment (EvilJackCarver) [inline code/modules/mob/living/carbon/superior_animal/stalker/stalker.dm:1]: I can't comment on the files themselves due to limitations of GitHub, but in the `code/modules/mob/living/carbon/superior_animal/stalker/` directory, `ai.dm`, `attack.dm` and `life.dm` are empty files. Is this intentional?

File: code/modules/projectiles/ammunition/bullets.dm [hunk 4]
```diff
@@ -515,7 +541,7 @@
 	projectile_type = /obj/item/projectile/bullet/cap
 
 /obj/item/ammo_casing/flare
-	name = "flare shell"
+	name = "AG FR \"Floodlight\""
```
Comment (EvilJackCarver) [inline code/modules/projectiles/ammunition/bullets.dm:544]: Aster's Guild abbreviation

File: code/modules/projectiles/guns/energy/plasma.dm [hunk 5]
```diff
@@ -45,7 +45,8 @@
 
 /obj/item/gun/energy/plasma/destroyer
 	name = "NT PR M \"Purger\""  //Eclipse Edit - gun names standardized
-	desc = "A more recent \"NanoTrasen\" brand plasma rifle, developed in direct response to compete against the highly successful \"Cassad\" design."
+	desc = "A more recent \"NanoTrasen\" brand plasma rifle, developed in direct response to compete against the highly successful \"Cassad\" design. \
+            There\'s an inscription on the stock. \'For those whom the Angels hath cursed: thou wilt find, have no one to help.\'"
```
Comment (EvilJackCarver) [inline code/modules/projectiles/guns/energy/plasma.dm:49]: Oddly religious to be coming from NanoTrasen...

**Review by EvilJackCarver — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/superior_animal/stalker/stalker.dm (hunk 3 above)
Comment (drexample) [reply code/modules/mob/living/carbon/superior_animal/stalker/stalker.dm:1]: apparently

File: code/modules/projectiles/guns/energy/plasma.dm (hunk 5 above)
Comment (drexample) [reply code/modules/projectiles/guns/energy/plasma.dm:49]: It's a donated weapon.

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 6]
```diff
@@ -82,24 +216,20 @@
...
-		/datum/design/bioprinter/engineering_tools,
-		/datum/design/bioprinter/engineering_supply,
-		/datum/design/bioprinter/engineering_material,
-		/datum/design/bioprinter/ammo,
-		/datum/design/bioprinter/tubular,
-		/datum/design/bioprinter/tubular/vial,
-		/datum/design/bioprinter/part,
+// Basic melee
+/obj/item/computer_hardware/hard_drive/portable/design/nt_melee
+	disk_name = "NeoTheology Armory - \"Hastatii Arms\""
```
Comment (EvilJackCarver) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:224]: Missed two, here's the first...

File: code/game/objects/items/weapons/design_disks/NeoTheology.dm [hunk 7]
```diff
@@ -342,6 +342,17 @@
 	disk_name = "Mekhane Armory - Cruciform Upgrades"
 	icon_state = "neotheology"
 
+/obj/item/computer_hardware/hard_drive/portable/design/guns/nt_purger
+	disk_name = "NeoTheology Armory - Purger Plasma Rifle"
```
Comment (EvilJackCarver) [inline code/game/objects/items/weapons/design_disks/NeoTheology.dm:346]: ...here's the second.

**Review by EvilJackCarver — CHANGES_REQUESTED** [review]
Just a couple missed NeoTheo references.

**Review by EvilJackCarver — APPROVED** [review]

**Pull request merged**